docstr_coverage/coverage.py

```diff
@@ -107,6 +108,9 @@ def get_docstring_coverage(
     skip_class_def: Boolean, default=False
         If True, skips class definitions and does not include them in the report.
         If this is True, the class's methods will still be checked
+    skip_private: Boolean, default=False
+        If True, skips function definitions beginning with an underscore and does
```

Comment:
Can you specify that it skips definitions beginning with "a single underscore" to avoid confusion with double underscore prefixes from magic methods or mangled names, please?

Reply:
Good point. I will also have to change the implementation to make sure it doesn't also automatically skip dunder functions

Reply:
Fixed in 79aa73f